src/summaryComponent.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+function SummaryComponent(storyUrl) {
+  this.storyUrl = storyUrl;
+  this.text = "burn your shoes";
```

Comment:
Would be good to discuss why we need this.text / how it will change once code develops